### DISCUSSION TITLE
Flightradar24 1.0.34-0

### DISCUSSION THREAD

**Discussion by misko85** [ketilmo/balena-ads-b#74] [Q&A] [answered]
When is an update to F24 1.0.34-0 planned?

**Comment by ketilmo** [comment, marked as answer]
Hi @misko85,

Thanks for reaching out.

The fr24feeder application has now been updated to version 1.0.34-0, as per commit f96512e2625a17f05f23d7b2b52f5d31261674ba. To roll this out to your devices, simply [update](https://github.com/ketilmo/balena-ads-b#part-13--updating-to-the-latest-version) your balena application. 

Feel free to let me know if you have any more questions. 

Thanks for using balena-ads-b!
